### PR TITLE
Improve README with top-level design goals and definition

### PR DESCRIPTION
**Pull request by brandon-powers**
### Why

We want to document some aspects of this project.

#3  

### What

- [x] Adds a Travis CI build status badge
- [x] Adds a description of what `apod-explorer` is and the two milestones

### CODE REVIEW

**Comment by mgzwarrior** [comment]
@brandon-powers - you can use [this link](https://travis-ci.com/mgzwarrior/apod-explorer.svg?branch=master) to add the status of the Travis CI build of the master branch of this repository to the README file.

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
-# apod-explorer
\ No newline at end of file
+### apod-explorer
+
+`apod-explorer` is an application that serves two primary purposes:
+
+- as a GraphQL-backed API, exposing data access to asynchronously download in bulk high-definition APOD photos
+- as a data visualization UI that enables exploration of APODs using a chosen date range; it'll dynamically download APODs if it's missing the range
```
Comment (mgzwarrior) [inline README.md]: Also, you can further clarify that the API work belongs to the v1 GitHub milestone and the UI front end work belongs to the v2 GitHub milestone.

Comment (brandon-powers) [reply README.md]: Good idea, done 👍.

**Comment by mgzwarrior** [comment]
@brandon-powers - I see you added some context to the description of this PR.  Let's open a new issue in the Infrastructure project to create a GitHub PR template for this repo to better guide development.

**Pull request closed without merging**